lib/src/signed_video_h26x_auth.c

```diff
@@ -332,28 +336,17 @@ verify_hashes_with_hash_list(signed_video_t *self,
     // Check if this is the item right after the |sei|.
     found_item_after_sei = (item->prev == sei);
     // Check if this |is_first_nalu_in_gop|, but not used before.
-    found_next_gop = (item->nalu->is_first_nalu_in_gop && !item->need_second_verification);
+    found_next_gop = (item->nalu->is_first_nalu_in_gop && !item->used_for_linked_hash);
```

Comment:
Can you give an example where `item->nalu->is_first_nalu_in_gop` and `!item->used_for_linked_hash` and `item->validation_status != 'P'`